activerecord/lib/active_record/destroy_job.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module ActiveRecord
+  class DestroyJob < ActiveJob::Base
```

Comment:
Yeah, I’d prefer to raise if no destroy job is configured.

Reply:
I think I agree with @georgeclaghorn, raise and emit a message that this is not supported without the inclusion of ActiveJob? I think it is misleading to complete a behavior that is not available in a different way, and if you have a noisy app you may never see a warning message. Essentially it would be like progressive enhancement in a web browser, you can still use active record but not every feature is available to you unless you bring in ActiveJob.

Reply:
I'm ok with that as well. It just that it doesn't make sense to raise if we can actually complete the work. I see this more as the inline adapter to Active Job than as progressive enhancement.